### PR TITLE
add no-cache headers

### PR DESCRIPTION
**Pull request by madhur-tandon**

### CODE REVIEW

**Comment by WebReflection** [comment]
the MR looks good but is there any reason no description is presented? I've no idea what I am approving otherwise, thanks for any sort of clarification, this header is hostile for mobile data too so I'd love to have at least a hint around why this is needed.

File: src/pyscript/plugins/run.py [hunk 1]
```diff
@@ -34,6 +34,7 @@ def end_headers(self):
             self.send_header("Cross-Origin-Opener-Policy", "same-origin")
             self.send_header("Cross-Origin-Embedder-Policy", "require-corp")
             self.send_header("Cross-Origin-Resource-Policy", "cross-origin")
+            self.send_header("CacheControl", "no-cache, must-revalidate")
```
Comment (FabioRosado) [inline src/pyscript/plugins/run.py:37]: Should this be `Cache-Control`?

**Review by FabioRosado — COMMENTED** [review]
@madhur-tandon  Do you remember why we need this? We can now merge stuff

File: src/pyscript/plugins/run.py (hunk 1 above)
Comment (WebReflection) [reply src/pyscript/plugins/run.py:37]: [Yes](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control)

Comment (FabioRosado) [reply src/pyscript/plugins/run.py]: pushed a fix

**Review by FabioRosado — APPROVED** [review]

**Pull request merged**